repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/block-links.ftl

```diff
@@ -0,0 +1,42 @@
+<#macro blockLinks block>
+    <#if block??>
+        <#if block.blockLinksContentBlock.blockLinks?size gt 0>
+            <section class="nhsuk-section">

```

Comment:
Since, this block is already being rendered within a section on guidance and blog post pages, I think section and it's associated div's aren't required.

I think the following section wrapper tags can be removed along with it's corresponding closing tags:
```
<section class="nhsuk-section">
  <div class="nhsuk-width-container">
    <div class="nhsuk-grid-row">
      <div class="nhsuk-grid-column-full nhsuk-section__content">
```

Let me know what you think ?

Reply:
Yes. To follow other examples, I've added `<div class="nhsuk-block-links">` instead. Commited on 548764f

Reply:
Thanks @Raq-HEE. It's nice that you have wrapped the whole block with `<div class="nhsuk-block-links">`, but the class `nhsuk-block-links` doesn't seem to exist in the CSS. Not sure if this is fine from the FE best practices perspective though.